pocs/oauth-server/account/verification.js

```diff
@@ -0,0 +1,20 @@
+const crypto = require('crypto')
+const mailer = require('../mailer')
+
+function prepareToken(email, VerifyModel = require('../db/schema/verify')) {
+    const token = crypto.randomBytes(64).toString('hex')
+
+    const data = {
+        email,
+        createdOn: new Date().getTime(),
+        token
+    }
+
+    const instance = new VerifyModel(data)
+
+    instance.save()
+    .then(() => mailer.sendVerificationMail(email, token))
+    .catch(console.log)
```

Comment:
I believe it won't work in the FF, could you ensure, whether it works in NodeJS?